crashtracker-ffi/Cargo.toml

```diff
@@ -10,12 +10,19 @@ license.workspace = true
 
 [lib]
 bench = false
+crate-type = ["lib", "cdylib"]
+
+[[bin]]
+name = "test_app"
+path = "tests/test_app/src/main.rs"
```

Comment:
What happens with this on Linux?

Reply:
The test application will compile but be empty.